src/main/java/org/broadinstitute/consent/http/models/DarCollection.java

```diff
@@ -144,6 +148,16 @@ public void addDar(DataAccessRequest dar) {
     }
   }
 
+  public DataAccessRequest getMostRecentDar() {
+    List<DataAccessRequest> dars = new java.util.ArrayList<>(
+        this.getDars().values().stream().toList());
+    dars.sort(Comparator.comparing(DataAccessRequest::getSubmissionDate).reversed());
+    if (dars.isEmpty()) {
+      return null;
+    }
+    return dars.get(0);
```

Comment:
You can pass a comparator to `Stream.sorted()`. Also since you want the first element or null, you can get the last element or null, without reversing. Although you can get the min or max value directly from a stream with a comparator and avoid sorting it.
```suggestion
    return dars.values().stream()
        .max(Comparator.comparing(DataAccessRequest::getSubmissionDate))
        .orElse(null);
```